apps/SageAccountWeb/src/AppInitializer.tsx

```diff
@@ -192,20 +202,25 @@ function AppInitializer(props: { children?: React.ReactNode }) {
         signedToken,
       }}
     >
-      <SynapseContextProvider
-        synapseContext={{
-          accessToken: token,
-          isInExperimentalMode: SynapseClient.isInSynapseExperimentalMode(),
-          utcTime: SynapseClient.getUseUtcTimeFromCookie(),
-          downloadCartPageUrl: '',
-        }}
-        theme={themeOptions}
+      <TwoFactorAuthSSOContextProvider
+        context={{ twoFactorAuthErrorResponse: twoFactorAuthError }}
       >
```

Comment:
Needed to add `TwoFactorAuthSSOContext` because

- `useDetectSSOCode` gets the data we need to complete SSO sign in with 2FA. This is called from `AppInitializer`.

- The `LoginPage` components that need to receive the code are created by `App`, which renders both `LoginPage` as a child of `AppInitializer`

Since `AppInitializer` doesn't know anything about `LoginPage`, the only way to pass the data from `AppInitializer` down into `LoginPage` is through context. There are probably ways we could refactor this to avoid this indirection, but I'm not yet very familiar with this code base, so I'm not equipped to make that kind of change.